pages/settings/interface/index.vue

```diff
@@ -14,6 +41,10 @@ useHeadFixed({
       </div>
     </template>
     <div p6 flex="~ col gap6">
+      <div v-if="loremStatus" border="~ base rounded" bg-active>
+        <StatusCard :status="loremStatus" />
```

Comment:
action buttons should be disabled or hidden.

Reply:
Perhaps we could add a disabled property to the status card that would disable interactions with it?

Reply:
Sure!